services/web/src/index.js

```diff
@@ -10,8 +10,11 @@ import { SENTRY_DSN } from 'utils/env';
 import { SessionProvider } from 'stores';
 import App from './App';
 
-if (SENTRY_DSN && window.Sentry) {
-  window.Sentry.init({ dsn: SENTRY_DSN });
+if (SENTRY_DSN) {
```

Comment:
if we're going this far I would prefer to have `index.js` load a module like `utils/sentry` so it's in one place and easier to turn off when not needed (the entrypoint tends to turn into spaghetti in a large project)

Reply:
I am not sure i understand? 
you want me to move these 2 lines of code into its own file ?

Reply:
well more than that right? minimally:

```js
import Sentry from '@sentry/sentry';
import { SENTRY_DSN, ENV_NAME } from 'utils/env';

if (SENTRY_DSN) {
  Sentry.init({
    dsn: SENTRY_DSN,
    environment: ENV_NAME,
  });
}
```

other projects have used a `captureError` method as well so probably will end up being more than that.

would be nice to just have this in `index.js`:

```
import 'utils/sentry';
```

Reply:
kk sure that doesn't seems too bad.

Is there projects that isnt using sentry? Is there other candidates? 

Just want to get the full picture or is also related running offline?

Reply:
I'm not sure what "offline" means, but the point is that many projects won't be paying for sentry

Reply:
3rd party services are questionable to be enabled by default at all, but minimally they should be easy to turn off

Reply:
done